### PR TITLE
docs(plugin): add docs for installing comment.nvim with lazy.nvim

### PR DESCRIPTION
**Pull request by sadikeey**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -17,6 +17,21 @@
...
+- With [lazy.nvim](https://github.com/folke/lazy.nvim)
+
+```lua
+return {
+    'numToStr/Comment.nvim',
+    keys = { -- example keymaps for lazy loading this plugin.
+      { "gcc", mode = "n", desc = "Toggles the current line using linewise comment" },
+      { "gbc", mode = "n", desc = "Toggles the current line using blockwise comment" },
+      { "gc", mode = "v", desc = "Toggles the region using linewise comment" },
+      { "gb", mode = "v", desc = "Toggles the region using blockwise comment" },
```
Comment (numToStr) [inline README.md:29]: That is way too much noise for an install instruction.

Comment (ryanmsnyder) [reply README.md:29]: What is the recommended way to install via lazy.nvim?

Comment (technicalpickles) [reply README.md:29]: The intent of `keys` here is to register the key mapping, but not load the plugin until the keys are actually use (ie 'lazy' load it).

If you don't include `keys`, then the next best thing would be using an `event` to use.

Comment (gegoune) [reply README.md:29]: Why would you want to lazy load this https://github.com/numToStr/Comment.nvim/blob/master/plugin/Comment.lua at all?

Comment (technicalpickles) [reply README.md:29]: That file specifically, or comment.nvim in general?

I am looking at that file, and I'm kinda surprised to not see a `setup` method 🤔 Actually, it looks like https://github.com/numToStr/Comment.nvim/blob/master/lua/Comment/init.lua is what would actually be loaded via `require('Comment').setup`. 

Lazy.nvim has a convention of, if you have `opts = true` or `opts = { ... }`, it essentially ends up doing:

```lua
require('Comment').setup(opts)
```

**Pull request closed without merging**